test/test.py

```diff
@@ -617,6 +619,11 @@ def testTags(self):
                 res = client.search(q)
                 self.assertEqual(1, res.total)
 
+                q2 = client.tagvals('tags')
```

Comment:
Let's add another document with additional words and see that we get the combined list.